src/translations/fr/importParamsFr.ts

```diff
@@ -0,0 +1,131 @@
+    'iidm.import.xml.extensions.hvdcOperatorActivePowerRange': 'Limites de transits des HVDCs',
+    'iidm.import.xml.extensions.identifiableShortCircuit': 'Données de court-circuit des postes',
+    'iidm.import.xml.extensions.injectionObservability': 'Observabilité des injections',
+    'iidm.import.xml.extensions.lineAsymmetrical': 'Données pour les calculs dissymétriques des lignes',
+    'iidm.import.xml.extensions.linePosition': 'Coordonnées géographiques de lignes',
+    'iidm.import.xml.extensions.loadAsymmetrical': 'Données pour les calculs dissymétriques des consommations',
+    'iidm.import.xml.extensions.measurements': 'Télémesures',
+    'iidm.import.xml.extensions.mergedXnode': 'Xnode mergé',
+    'iidm.import.xml.extensions.operatingStatus': 'Statut de consignation et de déclenchement',
+    'iidm.import.xml.extensions.position': 'Position des départs',
+    'iidm.import.xml.extensions.referencePriorities':
+        'Indice de priorité des noeuds de référence (Calcul de répartition)',
+    'iidm.import.xml.extensions.referenceTerminals': 'Terminaux référence de phase (Calcul de répartition)',
+    'iidm.import.xml.extensions.secondaryVoltageControl': 'Réglage secondaire de tension',
+    'iidm.import.xml.extensions.slackTerminal': 'Noeud bilan',
+    'iidm.import.xml.extensions.standbyAutomaton': 'Automate des CSPRs',
+    'iidm.import.xml.extensions.startup': 'Coût de démarrage des groupes',
+    'iidm.import.xml.extensions.substationPosition': 'Coordonnées géographiques des sites',
+    'iidm.import.xml.extensions.threeWindingsTransformerAsymmetrical':
+        'Données pour les calculs dissymétriques des transformateurs à trois enroulements',
+    'iidm.import.xml.extensions.threeWindingsTransformerPhaseAngleClock':
+        "Angles de phase entre les enroulements (sous forme d'horloge) pour les transformateurs à trois enroulements",
+    'iidm.import.xml.extensions.threeWindingsTransformerToBeEstimated':
+        'Estimation des prises des régleurs et des déphaseurs des transformateurs à trois enroulements',
+    'iidm.import.xml.extensions.twoWindingsTransformerAsymmetrical':
+        'Données pour les calculs dissymétriques des transformateurs à deux enroulements',
+    'iidm.import.xml.extensions.twoWindingsTransformerPhaseAngleClock':
+        "Angle de phase entre les enroulements (sous forme d'horloge) pour les transformateurs à deux enroulements",
+    'iidm.import.xml.extensions.twoWindingsTransformerToBeEstimated':
+        'Estimation des prises des régleurs et des déphaseurs des transformateurs à deux enroulements',
+    'iidm.import.xml.extensions.voltageLevelShortCircuits': 'Données de court-circuit des postes (Version IIDM 1.0)',
+    'iidm.import.xml.extensions.voltagePerReactivePowerControl': 'Contrôle de la tension par le réactif',
+    'iidm.import.xml.extensions.voltageRegulation': 'Réglage de tension',
+    'iidm.import.xml.extensions.xnode': 'Code Xnode',
```

Comment:
Same here :the english file is 125 lines long. Maybe it should be completed ?

Reply:
ok

Reply:
Indeed sorry. I got confused.